galadriel/dashboard/state.py

```
@@ -1,146 +1,193 @@
+MAX_LINKED_BUGS = 5
+
+
+def _fetch_issue(issue_key: str) -> dict | None:
+    """Fetch a single Jira issue. Designed to run in a thread pool."""
+    return jira.get_issue(issue_key)
+
+
 class DashboardState(rx.State):
 
-    linked_bugs: List[str] = []
+    linked_bugs: List[List[str]] = []
```

Ruff (0.15.4)

[warning] Mutable default value for class attribute

(RUF012)